modules/sdk/pages/overview.adoc

```diff
@@ -147,8 +147,56 @@ xref:2.1@go-sdk:hello-world:start-using-sdk.adoc[Go SDK 2.1]
 SDK doctor is a tool to diagnose application-server-side connectivity issues with your Couchbase Cluster.
 xref:sdk:sdk-doctor.adoc[SDK doctor]
```

Comment:
Should there be a blank line before this one? cf the **Java Transactions** and **C# Transactions** sections below.

Reply:
No, that's fine - it'll come out like the page at https://docs-staging.couchbase.com/home/sdk.html